### PR TITLE
improved welcome message

### PR DESCRIPTION
**Pull request by holzit**
changed the welcome message to be more informative regarding issue #42 

todo
- [ ] visual customization to resemble the readme

### CODE REVIEW

**Comment by fuerbringer** [comment]
I'm assuming this is related to issue #42, correct?

**Comment by holzit** [comment]
updated title and comment to reflect that it belongs to issue #42

File: public/js/core.js [hunk 1]
```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Welcome to Tarchivepannel!",
-			text: "Enter the key provided by the bot:",
+			 html:true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link:' +
+							'<a href="https://t.me/Tarchivebot"> Tarchivebot</a><br>' +
```
Comment (fuerbringer) [inline public/js/core.js]: I'm assuming this Link is clickable without any issues, right? If so, I'd probably add the parameter `target="_blank"`. Wouldn't want the panel to close on you ;)

Comment (holzit) [inline public/js/core.js]: > I'm assuming this Link is clickable without any issues, right?

link tested in chrome/firefox/safari on mac behaviour is as intended 

> I'd probably add the parameter target="_blank".

oops!

**Comment by fuerbringer** [comment]
@holzit Thank you. Looks good to me. Can we get another review from @Hoi15A ?

File: public/js/core.js [hunk 2]
```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Enter your key!",
-			text: "Enter the key provided by the bot:",
+			html: true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link: ' +
+							'<a href="https://t.me/Tarchivebot" target="_blank">Tarchivebot</a><br>' +
```
Comment (Hoi15A) [inline public/js/core.js]: While the link is completely fine in our situation if anyone else decides to run this bot the link will point to the wrong place.

As far as I'm aware there is a variable in the backend that is exported to be used in the pug, so you could have it be accessible like this if I recall correctly:
```pug
script.
    var botLink = {accessibleVar}
```

@fuerbringer probably knows a bit more on this.

File: public/js/core.js [hunk 3]
```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
...
+							'The bot requires read access to messages! <br><br>' +
+							'To retrieve your key simply type /apikey into chat<br><br>' +
```
Comment (Hoi15A) [inline public/js/core.js]: Maybe try putting`/apikey` into a `<pre>` tag if it looks good. Not sure if it will in practice though.

File: public/js/core.js (hunk 2 above)
Comment (fuerbringer) [inline public/js/core.js]: Ah yes, I totally forgot about this. Ideally certain parts of the welcome dialog should indeed be read out of the [config file](https://github.com/PROGRADE-Tech/Tarchivebot/blob/master/config/config.json.example#L4).

This would require passing the contents over to Express, Pug, and lastly the core.js script. However I'd be totally fine with having that solved in another PR, thoughts @Hoi15A ?

Comment (Hoi15A) [inline public/js/core.js]: I'd be fine with this being in another PR unless @holzit want's to do it directly in this PR already.

File: public/js/core.js (hunk 3 above)
Comment (holzit) [inline public/js/core.js]: image

**Comment by fuerbringer** [comment]
Solves #42
